consensus/config/src/committee.rs

```diff
@@ -126,6 +124,53 @@ impl Committee {
     pub fn size(&self) -> usize {
         self.authorities.len()
     }
+
+    pub fn elect_leader(&self, round: u32, leader_offset: u32) -> AuthorityIndex {
```

Comment:
Let's put leader selection in a separate struct and file e.g. leader_schedule.rs. This will save time when integrating leader schedule change.